python-avd/pyavd/_eos_designs/structured_config/base/__init__.py

```diff
@@ -101,7 +101,7 @@ def router_bgp(self) -> dict | None:
             )
 
         l3_interfaces_neighbors = []
-        for neighbor_info in self.shared_utils.l3_interfaces_bgp_neighbors:
+        for neighbor_info in [*self.shared_utils.l3_interfaces_bgp_neighbors, *self.shared_utils.l3_port_channels_bgp_neighbors]:
```

Comment:
Do we ever need the bgp_neighbors on their own for each type? Maybe it would be simpler to just have an `l3_bgp_neighbors` containing both instead of repeatedly unpacking and repacking the lists.

Reply:
Yes, we now have single property `l3_bgp_neighbors` that represents consolidated BGP neighbors across both L3 interfaces and L3 Port-Channels. Have left `l3_interfaces_bgp_neighbors` property as-is for now. The callers now refer to `l3_bgp_neighbors` property instead.